atlp-rwanda/knights-ecomm-fe
# fixes hard coded data on vendor products page

#### What does this PR do?
This PR fixes a issue on the vendor product page, ensuring that the data is accurate, the page is responsive on all devices,

#### Description of Task to be completed?
- Ensure data displayed on the page are accurate and dynamic according to data in db.
- Ensure it looks good on all devices (responsiveness).
- Add a message to inform users about what is happening while the page is loading

#### How should this be manually tested?
- Ensure You have backend service running and set VITE_APP_API_URL in your .env file.
- Clone this repository and navigate to the project directory
  ```bash
  git clone -b ft-register https://github.com/atlp-rwanda/knights-ecomm-fe.git
  cd knights-ecomm-fe
  git checkout fix-vendor-product
  npm i
  npm run dev
  ```
